### PR TITLE
Alert: Redesign with tinted background 

### PR DESCRIPTION
**Pull request by torkelo**
Our current inline banners just look so "meh", and I have been wanting to move to a move "transparent" and common design for years. 

Current: 
Screenshot from 2023-04-20 10-22-20
Screenshot from 2023-04-20 10-22-36
Screenshot from 2023-04-20 10-22-49

New design (in collaboration with Staton and Erik) 

Screenshot from 2023-04-28 16-40-07

Screenshot from 2023-04-28 16-15-42

Screenshot from 2023-04-28 16-40-12

Screenshot from 2023-04-28 16-40-26

Screenshot from 2023-04-28 16-50-20

Screenshot from 2023-04-28 16-47-54

### CODE REVIEW

**Comment by torkelo** [comment]
Here is an option with aligning the icon to the top / title instead of vertically centered 
Screenshot from 2023-04-20 10-40-48
Screenshot from 2023-04-20 10-40-36

**Comment by torkelo** [comment]
@staton-hyse11 pushed the latest changes, must rush to school now!

**Comment by torkelo** [comment]
Here is an example of how it really improves large info boxes: 

Screenshot from 2023-04-28 17-31-54

Screenshot from 2023-04-28 17-32-48

File: packages/grafana-ui/src/components/Alert/Alert.tsx [hunk 1]
```diff
@@ -73,7 +73,14 @@ export const Alert = React.forwardRef<HTMLDivElement, Props>(
         {/* If onRemove is specified, giving preference to onRemove */}
         {onRemove && !buttonContent && (
           <div className={styles.close}>
-            <IconButton aria-label="Close alert" name="times" onClick={onRemove} size="lg" type="button" />
+            <Button
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/Alert/Alert.tsx:76]: What was the reason for changing to `Button`? Is it just the "squareness" of it? Just curious for our own investigation of unifying these two options

File: packages/grafana-ui/src/components/Alert/Alert.tsx [hunk 2]
```diff
@@ -113,6 +120,7 @@ const getStyles = (
 ) => {
   const color = theme.colors[severity];
   const borderRadius = theme.shape.borderRadius();
+  const borderColor = tinycolor2(color.border).setAlpha(0.2).toString();
```
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/Alert/Alert.tsx:123]: This just made me realize that we actually use `setAlpha` in a bunch of places, so we need to remember this during the color tokens exploration so we can take these into account.

File: public/app/features/connections/components/ConnectionsRedirectNotice/ConnectionsRedirectNotice.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
   alertParagraph: css`
     margin: 0 ${theme.spacing(1)} 0 0;
     line-height: ${theme.spacing(theme.components.height.sm)};
-    color: ${theme.colors.text.primary};
+    //color: ${theme.colors.text.primary};
```
Comment (JoaoSilvaGrafana) [inline public/app/features/connections/components/ConnectionsRedirectNotice/ConnectionsRedirectNotice.tsx:20]: To delete?

**Review by JoaoSilvaGrafana — COMMENTED** [review]
These do look so much better, nice job! 👏

File: packages/grafana-data/src/themes/createColors.ts [hunk 4]
```diff
@@ -280,7 +280,7 @@ export function createColors(colors: ThemeColorsInput): ThemeColors {
       color.shade = base.mode === 'light' ? darken(color.main, tonalOffset) : lighten(color.main, tonalOffset);
     }
     if (!color.transparent) {
-      color.transparent = base.mode === 'light' ? alpha(color.main, 0.08) : alpha(color.main, 0.15);
+      color.transparent = base.mode === 'light' ? alpha(color.main, 0.15) : alpha(color.main, 0.15);
```
[the remark this answers is not in the document]
Comment (torkelo) [reply packages/grafana-data/src/themes/createColors.ts:283]: good point :)

File: packages/grafana-ui/src/components/Alert/Alert.tsx (hunk 1 above)
Comment (torkelo) [reply packages/grafana-ui/src/components/Alert/Alert.tsx:76]: @JoaoSilvaGrafana yea, two reasons the squareness (which looks good when it matches the shape of the corner it's in) and the color of the hover effect for a secondary fill button is transparent so it takes on the color tint of the background which looks really good.

File: packages/grafana-ui/src/components/Alert/Alert.tsx (hunk 2 above)
Comment (torkelo) [reply packages/grafana-ui/src/components/Alert/Alert.tsx:123]: @JoaoSilvaGrafana yes, if we need this weak colored border in many places it should be added as a token in theme. but so far this and the Badge is are the only two I know of we do this with the border color .

**Review by staton-hyse11 — APPROVED** [review]
LGTM! I've created a backlog item for the follow-up tasks [here](https://github.com/grafana/grafana/issues/67587)

**Comment by teddyba** [comment]
I love the new design -- I think it draws attention to the information without throwing off the visual balance of the page. We actually used this style with the transparent colored background in a few places already in Cloud products, because the traditional banner style was too overwhelming. 

Great job!!

**Comment by torkelo** [comment]
thanks @teddyba , yea I really think it makes the UI look a bit more fresh and soft. Not as hard and solid :)

**Pull request merged**